src/index.html

```diff
@@ -7,12 +8,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <img
+        src="../src/images/imac.png"
+        alt="imac"
+      />
+
+      <div class="card__t">
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
```

Comment:
The class `card__t` seems to be a typo or not clearly named. It's important for class names to be descriptive. Consider renaming it to `card__title-container` or a similar descriptive name.